### DISCUSSION TITLE
jar missing for release 0.3.5

### DISCUSSION THREAD

**Discussion by karussell** [komoot/photon#596] [General]
In https://github.com/komoot/photon/releases/tag/0.3.5 the jar is not yet uploaded. I think we manually uploaded this for older releases: https://github.com/komoot/photon/releases/tag/0.3.4 or maybe this release creation should be automated via github CI and github packages?

**Comment by lonvia** [comment]
Right, I forgot to upload the jar. Fixed.

It shouldn't be too hard to do an automated packaging for releases. See [nominatim-ui](https://github.com/osm-search/nominatim-ui/blob/master/.github/workflows/release.yml) for an example.

**Reply by karussell** [reply]
Thanks for the quick fix!
